docs/manual/oidc.rst

```diff
@@ -9,7 +9,7 @@ Open Notificaties ondersteunt Single Sign On (SSO) via het OpenID Connect protoc
 Gebruikers kunnen op die manier inloggen op Open Notificaties met hun account bij de OpenID Connect provider. In deze
 flow:
 
-1. Klikt een gebruiker op het inlogscherm op *Inloggen met organisatieaccount*
+1. Klikt een gebruiker op het inlogscherm op *Inloggen met OIDC*
```

Comment:
Was this translation changed perhaps? Because in the template I still see "Login with organization account" as the English label

Reply:
I haven't updated translations yet, I'll have to double check that! :smiley:

Reply:
you were right :-) I've updated the translations now so that docs and UI match and reverted the docs change